apps/omg_status/mix.exs

```diff
@@ -35,11 +35,11 @@ defmodule OMG.Status.Mixfile do
     do: [
       {:telemetry, "~> 0.4.1"},
       {:sentry, "~> 7.0"},
-      {:statix, git: "https://github.com/omisego/statix.git", branch: "otp-21.3.8.4-support-global-tag-patch"},
+      {:statix, git: "https://github.com/omgnetwork/statix.git", branch: "otp-21.3.8.4-support-global-tag-patch"},
       {:spandex_datadog, "~> 0.4"},
       {:decorator, "~> 1.2"},
       {:vmstats, "~> 2.3", runtime: false},
-      {:ink, "~> 1.1"},
+      {:ink, git: "https://github.com/omgnetwork/ink.git", branch: "unnawut/full-log-on-error"},
```

Comment:
I saw this comment https://github.com/ivx/ink/pull/63#issuecomment-650132999 . Does it mean this PR is not final?

Reply:
So this is from the discussion at the last weekly sync to have the changes applied and running on our fork first. Then apply the PR upstream